game-app/game-core/src/main/java/games/strategy/triplea/ui/ObjectiveDummyDelegateBridge.java

```diff
@@ -85,13 +85,12 @@ public Player getRemotePlayer() {
       final DiceType diceType,
       final String annotation) {
     if (count <= 0) {
-      throw new IllegalStateException("count must be > o, annotation:" + annotation);
+      throw new IllegalStateException("count must be > 0, annotation:" + annotation);
     }
-    final int[] numbers = new int[count];
-    for (int i = 0; i < count; i++) {
-      numbers[i] = getRandom(max, player, diceType, annotation);
-    }
-    return numbers;
+    // Oracle documentation: "Each class variable, instance variable, or array component
+    // is initialized with a default value when it is created (§15.9, §15.10)
+    // ... For type int, the default value is zero, that is, 0."
+    return new int[count];
```

Comment:
Why is this returning an array of all 0's instead of random numbers?

Reply:
Oh, because this is a Dummy class?

Anyway, I'm not sure a comment quoting the Oracle docs is needed, you can just say "Return an array of all 0's."

I think it's reasonable to expect people writing Java code to know what the default values of variables are.

Reply:
Sure? That was not certain in the past if I remember correctly and you asked yourself.

Reply:
@asvitkine Is this discussion done for you?